packages/Adapters/ClassicDTK/src/DTK_VolumeSourceMap_def.hpp

```diff
@@ -171,7 +171,7 @@ void VolumeSourceMap<Geometry, GlobalOrdinal, CoordinateField>::setup(
         Teuchos::ArrayRCP<GlobalOrdinal> source_gids =
             source_geometry_manager->gids();
         int local_num_source_geom = source_geometry_manager->localNumGeometry();
-        for ( std::size_t i = 0; i < local_num_source_geom; ++i )
+        for ( int i = 0; i < local_num_source_geom; ++i )
```

Comment:
Shouldn't you change `local_num_source_geom`? It doesn't make sense for this to be negative.